frontend/package.json

```diff
@@ -40,16 +40,20 @@
   "license": "ISC",
   "devDependencies": {
     "babel-jest": "^23.4.2",
+    "babel-plugin-transform-imports": "^2.0.0",
     "babel-preset-env": "^1.7.0",
+    "clean-webpack-plugin": "^3.0.0",
     "enzyme": "^3.6.0",
     "enzyme-adapter-react-16": "^1.2.0",
     "enzyme-react-intl": "^2.0.2",
     "extract-react-intl": "^0.6.0",
     "extract-react-intl-messages": "^0.10.0",
     "fetch-mock": "^7.0.5",
+    "html-webpack-plugin": "^4.0.1",
     "jest": "^23.5.0",
     "jest-cli": "^23.6.0",
     "jest-webpack": "^0.5.1",
+    "mini-css-extract-plugin": "^0.9.0",
```

Comment:
When we try to deploy, Heroku logs the following error:
`Cannot find module 'mini-css-extract-plugin'`

So I think we cannot move here, cause Heroku use this lib in production.

Reply:
I have made the changes. I think your'e correct. Heroku doesn't install dev dependecies that is why all the plugins in our project our installed in package.json